RobotCode/src/main/java/frc/robot/Commands/ShooterCommand.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.Commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Subsystems.ShooterSubsystem;
+
+public class ShooterCommand extends CommandBase
+{
+	// makes the subsystem object
+	private final ShooterSubsystem Shooter;
+
+	public ShooterCommand(ShooterSubsystem shoot)
+	{
+		
+		Shooter = shoot;
+		addRequirements(Shooter);
+	}
+	// execute command
```

Comment:
Unnecessary comment: it restates the method name without adding any new information.

Reply:
Also be sure to have a blank line between methods!